javascript/README.md

```diff
@@ -90,6 +90,21 @@ Grammar supports the following list of ECMAScript 6 features taken from
 
 See [examples](examples) directory with test data files.
 
+
```

Comment:
But I meant to insert a line break after the header but not before :) Also, see [MD022](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md022).

Reply:
My mistake. Fixed.